Things.md

```diff
@@ -169,4 +170,10 @@ Events can also be added to repeat, similar to using `setInterval`.  A new third
 FSM.addEventInterval(FSM.shiftHoriz, 100, 5, thing, 2);
 ```
 
-More can be read about TimeHandlr on its [Readme](https://github.com/FullScreenShenanigans/TimeHandlr/blob/master/README.md).
\ No newline at end of file
+More can be read about TimeHandlr on its [Readme](https://github.com/FullScreenShenanigans/TimeHandlr/blob/master/README.md).
+
+# Thing classes
+
+Classes are implemented using [ObjectMakr](https://github.com/FullScreenShenanigans/ObjectMakr), a module that automates the creation of the inheritance structure of classes. The structure and a list of properties are given and the class objects are then constructed using this information. 
+
+When a FullScreenShenanigans game starts up, ObjectMakr receives the inheritance structure and properties list from an `objects.js` file. When this happens, it creates prototypes for all of the internal classes (Box, Tree, etc.). Most of the basic properties, like `height`, `width`, and `opacity` are defined in the parent Thing class. When a new Thing is created, it uses these pre-initialized prototypes to instantiate the new Thing object. 
```

Comment:
> creates prototypes

Very close - say it creates classes. That's more technically accurate, because a class *uses* (has) a prototype.